packages/manager/apps/pci-load-balancer/src/api/data/api-specs.ts

```diff
@@ -0,0 +1,20 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export type TApiSpecsResponse = {
+  models: {
+    'cloud.loadbalancing.PoolAlgorithmEnum': {
+      enum: string[];
+    };
+    'cloud.loadbalancing.PoolProtocolEnum': {
+      enum: string[];
+    };
+    'cloud.loadbalancing.PoolSessionPersistenceTypeEnum': {
+      enum: string[];
+    };
+  };
+};
+
+export const getApiSpecs = async (): Promise<TApiSpecsResponse> => {
+  const { data } = await v6.get<TApiSpecsResponse>(`/cloud.json`);
+  return data;
+};
```

Comment:
this could be put in pci-common as it is global to pci

also i would suggest naming it api schema instead of api specs

Reply:
was the actual name, but okey for the move and the rename